benchmark/checkperf.cmake

```diff
@@ -60,7 +60,7 @@ if (Git_FOUND AND (GIT_VERSION_STRING VERSION_GREATER  "2.1.4") AND (NOT CMAKE_G
   add_custom_command(
     OUTPUT ${SIMDJSON_CHECKPERF_DIR}/build/cmake_install.cmake # We make many things but this seems the most cross-platform one we can depend on
     COMMAND
-    ${CMAKE_COMMAND} -E env CXX=${CMAKE_CXX_COMPILER} CC=${CMAKE_C_COMPILER}
+    ${CMAKE_COMMAND} -E env CXX=${CMAKE_CXX_COMPILER} CC=${CMAKE_C_COMPILER} CI=$ENV{CI}
```

Comment:
I'm super puzzled: I see mention of this in the docs but can't find where we do if (CI) in our cmake code, either in this PR or in my tree.

Reply:
https://github.com/simdjson/simdjson/pull/1512/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR51-R53

The `env` shouldn't really be used here anyway, the variables could be passed as is via `-D` arguments, but in this PR my focus was clearly separating consumer and developer side of things.

Reply:
I'm going to unresolve this, because I was dumb when I wrote the code. `CI` is only defined in the CI and this target is behind the developer mode switch and operates on developer mode targets, so setting the developer mode flag unconditionally makes the most sense.

Reply:
How did this get resolved?

Reply:
You marked the conversation as resolved.

If you mean in the code, then it's here https://github.com/simdjson/simdjson/commit/083c3fb10d7f5100e37ff0c6b13968d3f935443c